src/deep-resource/test/Resource.js

```diff
@@ -4,180 +4,115 @@ import chai from 'chai';
 import sinon from 'sinon';
 import sinonChai from 'sinon-chai';
 import {Resource} from '../lib.compiled/Resource';
+import {Instance as ResourceInstance} from '../lib.compiled/Resource/Instance';
 import {MissingResourceException} from '../lib.compiled/Exception/MissingResourceException';
 import {Instance} from '../node_modules/deep-kernel/lib.compiled/Microservice/Instance';
 import Kernel from 'deep-kernel';
 import Cache from 'deep-cache';
 import Security from 'deep-security';
+import KernelFactory from './common/KernelFactory';
+import backendConfig from './common/backend-cfg-json';
```

Comment:
- there is no reason to import cfg file in order to create a microservice instance, you can get one already created from kernelInstance